package-res/resources/web/_activeVizTypes.js

```diff
@@ -14,9 +14,25 @@
  *
  * Copyright (c) 2012 Pentaho Corporation..  All rights reserved.
  */
-pen.define([
-    'common-ui/vizapi/VizController'
-], function() {
-    // Unfortunately couldn't configure this dependency through a shim and `init`. Didn't run.
-    return pentaho.VizController;
-});
\ No newline at end of file
+define([
+    "require",
```

Comment:
That's an interesting one. Requiring `require` returns what is called a contextualized require function. Unlike the global `require` function, that resolves relative paths in relation to `config.baseUrl`, the contextualized one resolves them in relation to where the `define` from which it was required.